app/views/cards/personaldetails/mainAddress.scala.html

```diff
@@ -28,7 +28,8 @@
       @address.lines.map { line =>
         @upperCaseToTitleCase(line)<br>
       }
-    @address.postcode.map(_.toUpperCase)
+    @address.postcode.map(_.toUpperCase)<br>
+    @if(!countries.contains(Country(address.country.map(_.toUpperCase).getOrElse("")))) {@address.country.map(_.toUpperCase)}
   }
   </strong></p>
```

Comment:
`<strong>` has semantic meaning. Is this only applied to provide bold? Or is it an important message